src/interface/meta.thrift

```diff
@@ -1271,9 +1272,9 @@ service MetaService {
     GetZoneResp    getZone(1: GetZoneReq req);
     ListZonesResp  listZones(1: ListZonesReq req);
 
-    ExecResp       addListener(1: AddListenerReq req);
-    ExecResp       removeListener(1: RemoveListenerReq req);
-    ListListenerResp listListener(1: ListListenerReq req);
+    ExecResp          addListener(1: AddListenerReq req);
+    ExecResp          removeListener(1: RemoveListenerReq req);
+    ListListenersResp listListeners(1: ListListenersReq req);
```

Comment:
Does the change of this interface affect the compatibility of different nebula version?